node-perf-dash/kubelet-parser.go

```diff
@@ -188,7 +194,7 @@ func GrabTracingKubelet(d Downloader, job string, buildNumber int, nodeName stri
 			"test": sortedTestTime[currentTestIndex].TestName,
 			"node": nodeName,
 		},
-		Version: tracingVersion,
+		Version: perftype.CurrentKubeletPerfMetricsVersion,
```

Comment:
This is because node-perf-dash itself also generates perf data logs by parsing the content of the `kubelet.log`.

E.g.,
```
[Result:TimeSeries]{"labels":{"node":"n1-standard-1-coreos-alpha-1122-0-0-v20160727-e4490a91","test":"resource_35"},"version":"v1","op_series":{"container_start_pleg":[1496646375134107000, ...
...
[Finish:TimeSeries]
```